php/libraries/TimePoint.class.inc

```diff
@@ -921,5 +921,39 @@ class TimePoint
         return $candidate->getCandidateDoB();
     }
 
-} // end class
+    /**
+     * Gets the Images of that session
+     *
+     * @param \User $user The requesting user
+     *
+     * @return \LORIS\ImageDTO[] A traversable of \LORIS\ImageDTO
+     */
+    public function getImages(\User $user): array
+    {
+        $provisioner = new \LORIS\api\VisitImagesRowProvisioner($this);
+        $traversable = (new \LORIS\Data\Table())
+            ->withDataFrom($provisioner)
+            ->getRows($user);
+
+        return iterator_to_array($traversable);
+    }
+
+    /**
+     * Gets the DicomTar of that session
```

Comment:
```suggestion
     * Gets the DicomTar of that session.
```

Reply:
again, a given session could have multiple DicomTars so this should be reflected in the documentation